tensorflow/core/kernels/data/reader_dataset_ops.cc

```diff
@@ -482,21 +568,29 @@ class FixedLengthRecordDatasetOp : public DatasetOpKernel {
       mutex mu_;
       size_t current_file_index_ GUARDED_BY(mu_) = 0;
       std::unique_ptr<RandomAccessFile> file_
-          GUARDED_BY(mu_);  // must outlive input_buffer_
-      std::unique_ptr<io::InputBuffer> input_buffer_ GUARDED_BY(mu_);
+          GUARDED_BY(mu_);  // must outlive buffered_input_stream_
+      std::unique_ptr<io::RandomAccessInputStream>
+          file_stream_;  // must outlive buffered_inputstream_
```

Comment:
Nit: s/buffered_inputstream_/buffered_input_stream_/ here and on the previous line.